.bithoundrc

```diff
@@ -42,17 +42,13 @@
     "**/extjs/src/**",
     "**/extjs/welcome/**",
     "bower_components/**",
-    "**/assets/**"
+    "**/assets/**",
+    "public/js/tracking/**"
```

Comment:
webtrends code moved into a public/js/vendor folder so we can be specific about not doing code analysis on 3rd party code.